jolly-jellyfish/src/static/themes/galaxy.css

```diff
@@ -0,0 +1,27 @@
+/**
+ * Galaxy GIF in background, images in corner.
+ * Green text with black border and white "3D" effect.
+ * Text fades in and out in very wide font.
+ * Requires "static/images/galaxy.png" and "static/images/galaxy.gif" static files.
+ */
+
+@import url('https://fonts.googleapis.com/css2?family=Lexend+Zetta&display=swap');
```

Comment:
Maybe we can accomplish this without having to resort to a web font? That is, if stretching is the only reason you are using this font...
(see: https://stackoverflow.com/questions/6351013/can-i-stretch-text-using-css)

Maybe this way we could make a CSS animation to *dynamically make text stretch*?

Reply:
Sure, we could do that, but is there a problem with using web fonts? In fact I just scrolled through Google fonts for one that looked like it would fit.

Reply:
None really, I was just wondering if we could maybe make text stretch dynamically which sounds like something that would be interesting at least to see.

Reply:
Yes, certainly "interesting"... Which is the whole point of these themes.

Load times and potential tracking by Google are the major issues I suppose. No significant load time difference for me on my low end laptop with Wi-Fi though... And we're already repainting the page constantly to adjust the opacity of the text lol.

But sure, try it out and send a few screenshots. I have already used Web fonts in other themes though...